onnx_tf/gen_status.py

```diff
@@ -20,67 +20,46 @@ def main(docs_dir):
   base_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
   docs_dir = os.path.join(base_dir, 'doc')
   onnx_version = ''
-  onnx_path = ''
   onnx_tf_release_build = False
 
   try:
-    opts, args = getopt.getopt(
-        sys.argv[1:], 'hv:p:r',
-        ['onnx_version=', 'onnx_path=', 'onnx_tf_release_build'])
+    opts, args = getopt.getopt(sys.argv[1:], 'hv:r',
+                               ['onnx_version=', 'onnx_tf_release_build'])
   except getopt.GetoptError:
-    print('gen_status.py -v <onnx_version> -p <onnx_path> ' +
-          '-r <onnx_tf_release_build>')
-    print('If onnx_version=master then onnx_path is required.')
+    print('gen_status.py -v <onnx_version> -r <onnx_tf_release_build>')
     print('eg. generate support_status.md for onnx-tf master and onnx master')
-    print('        gen_status.py -v master -p /YOUR-ONNX-PATH')
+    print('        gen_status.py -v master')
     print('    generate support_status_<onnx_tf_version>.md for onnx-tf ' +
           'version state in the VERSION_NUMBER file and onnx v1.5.0 ')
```

Comment:
Is it 'version state' or 'version stated'? If the latter, consider make it clearer, such as 'version that is specified'.